src/main/java/io/vertx/core/http/HttpServerRequest.java

```diff
@@ -71,6 +71,11 @@
    * @return the HTTP method for the request.
    */
   HttpMethod method();
+  
+  /**
+   * @return whether the HTTP connection is secure
```

Comment:
please use the same wording that in NetSocket : `true if this {@link io.vertx.core.net.NetSocket} is encrypted via SSL/TLS.`